nubis/Puppetfile

```diff
@@ -48,7 +48,7 @@ mod 'nubis-confd_site',
 
 mod 'srf/fluentd',
     :git => 'https://github.com/gozer/puppet-fluentd.git',
-    :ref => 'd3b744bd591b2c44169f664f53f888ad4dc33041'
+    :ref => 'df5b93d7dddf8f43b82dced9e27c52df6a270642'
```

Comment:
This is my repo, and unfortunately, hasn't been merged back upstream yet, so we must use it for now